content/logs/processing/attributes_naming_convention.md

```diff
@@ -0,0 +1,188 @@
+| `http.url_details.port`        | `number` | The HTTP port part of the url.                                                          |
+| `http.url_details.path`        | `string` | The HTTP path part of the url.                                                          |
+| `http.url_details.queryString` | `object` | The HTTP query string parts of the url decomposed as query params key/value attributes. |
+| `http.url_details.scheme`      | `string` | The protocol name of the URL (http or https)                                            |
+
+### User-Agent attributes
+
+Details about the meanings of user agents attributes. Generally generated thanks to the [User-Agent parser][14]. All attributes are prefixed by `http.useragent_details`.
+
+| **Fullname**                            | **Type** | **Description**                                |
+| :---                                    | :---     | :----                                          |
+| `http.useragent_details.os.family`      | `string` | The OS family reported by the user-agent.      |
+| `http.useragent_details.browser.family` | `string` | The Browser Family reported by the user-agent. |
+| `http.useragent_details.device.family`  | `string` | The Device family reported by the user-agent.  |
+
+## Source code
+
+Related to the data used when a log or an error is generated via a logger in a custom application. All attributes are prefixed either by `logger` or `error`.
+ 
+| **Fullname**         | **Type** | **Description**                                                  |
+| :---                 | :---     | :----                                                            |
+| `logger.name`        | `string` | The name of the logger.                                          |
+| `logger.thread_name` | `string` | The name of the current thread when the log is fired.            |
+| `logger.method_name` | `string` | The class method name.                                           |
+| `error.kind`         | `string` | The error type or kind (or code is some cases).                  |
+| `error.message`      | `string` | A concise, human-readable, one-line message explaining the event |
+| `error.stack`        | `string` | The stack trace or the complementary information about the error |
+
+Typical integrations relying on these attributes are: *Java*, *NodeJs*, *.NET*, *Golang*, *Python*, etc.
+
+## Database
+
+Database related attributes are prefixed by `db`.
+
+| **Fullname**   | **Type** | **Description**                                                                                                                       |
+| :---           | :---     | :----                                                                                                                                 |
+| `db.instance`  | `string` | Database instance name. E.g., In java, if the `jdbc.url="jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `customers`.   |
```

Comment:
| `db.instance`  | `string` | Database instance name. E.g., in Java, if `jdbc.url="jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `customers`.   |